packages/plugins/plugin-appsflyer/src/AppsflyerPlugin.tsx

```diff
@@ -35,4 +85,41 @@ export class AppsflyerPlugin extends DestinationPlugin {
     track(event);
     return event;
   }
+
+  conversionCallback = () => {
```

Comment:
The name of this function should be like `registerConversionCallback` as this isn't the callback itself but a function that registers it.

It's usually a good idea to start all the functions with a verb